Urania/Urania.Core/OdCalculator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace Urania.Core {
+    public class OdCalculator {
+        public static decimal Calculate(decimal? ar, decimal? wd, decimal? id) {
+
+
+            if (id < 0) {
+                throw new ArgumentOutOfRangeException($"{nameof(id)} cannot be less than 0");
```

Comment:
Prawidłowa implementacja tego wyjątku wygląda tak:
`throw new ArgumentOutOfRangeException(nameof(id), id, "Cannot be less than 0");`